### ISSUE TITLE
exclusion of IGRs < 30 bp

### ISSUE THREAD

**Issue by maesaar** [harry-thorpe/piggy#18]
Could you please explain the reason behind exclusion of IGRs < 30bp?

Please lable this as question!

**Comment by harry-thorpe** [comment]
I mainly did this because otherwise really short IGRs would cluster together by chance. Also, these short IGRs are likely to be spacers between very close genes, and are unlikely to carry regulatory elements.

Perhaps I should add an option to set min and max sizes?

**Comment by maesaar** [comment]
Thanks for explanation - min & max sizes could be useful.

**Comment by maesaar** [comment]
One more thing - Did you test minimal sample size to work with Piggy, which gives reasonable results? I tested different sample sets with different number (n=30; n=50; n=200; n=400; n=600) of isolatest. High number of isolates gave similar expected results as in the publication, but when I used small no of samples (n=30) I got results that differed alot. I was wondering if Piggy results can be influenced by small sample size?
